test/memory-store-test.js

```diff
@@ -4,7 +4,7 @@ const MemoryStore = require("../lib/memory-store.js");
 
 describe("MemoryStore store", () => {
   it("sets the value to 1 on first incr", (done) => {
-    const store = new MemoryStore(-1);
+    const store = new MemoryStore(-1, 4);
```

Comment:
Lets undo these changes and instead add some tests that specifically check different precision values.

```suggestion
    const store = new MemoryStore(-1);
```